gradle/libs.versions.toml

```diff
@@ -12,7 +12,7 @@ intellij = "2023.1.5"
 # IntelliJ version to run against (`runIde` task). Use latest version.
 # https://www.jetbrains.com/idea/download/#section=mac
 intellijRunIde = "2023.2"
-intellijPlugin = "1.15.0"
+intellijPlugin = "1.17.4"
```

Comment:
Unrelated: I was having difficulty viewing plugin SDK sources, and bumping this plugin version fixed the problem for me.